### PR TITLE
Feature: Basic admin dashboard

### PR DESCRIPTION
**Pull request by varffs**
- Makes the /admin base into a basic dashboard with some basic analysis of the data in the view
- Adds the list donations view

### CODE REVIEW

File: app/client/templates/admin/admin.html [hunk 1]
```diff
@@ -2,38 +2,72 @@
...
-          <div>
-            £{{amount}} <em>{{timeSince createdAt}}</em>
-          </div>
-        {{/each}}
-
+        oneoff donations in the last month
       </div>
       <div class="col col12">
-        <h3>Subscriptions</h3>
+        new subscription in the last month
```
Comment (mixcas) [inline app/client/templates/admin/admin.html]: I don't like unwrapped text. IMO should always be in a semantic tag like `<p>`.

File: app/client/templates/admin/admin.js [hunk 2]
```diff
@@ -1,3 +1,105 @@
+Template.admin.helpers({
```
Comment (mixcas) [inline app/client/templates/admin/admin.js]: I feel these functions repeat a lot of code and could be abstracted in a better way.

File: app/client/templates/admin/admin.html (hunk 1 above)
Comment (varffs) [inline app/client/templates/admin/admin.html]: Yeah fair enough I'll do that with h4's or something

File: app/client/templates/admin/admin.js (hunk 2 above)
Comment (varffs) [inline app/client/templates/admin/admin.js]: Yes I noticed that as well but didn't have any good ideas on how to abstract. Have you got any suggestions? Think with them is the variables are things like collection names which means the abstracted function has to do string matching on a variable name or something which feels ugly. But that aside it is probably still better to do that and keep what is common unduplicated

Comment (mixcas) [inline app/client/templates/admin/admin.js]: actually its more an illusion, they are pretty diff and trying to abstract them only seems likely but its rly not

**Comment by varffs** [comment]
@mixcas I think this is ready to go now

**Pull request merged**